pytest_xprocess.py

```diff
@@ -64,12 +55,40 @@ def pytest_runtest_makereport(item, call):
                 longrepr.addsection("%s log" % name, content)
 
 
+def pytest_unconfigure(config):
```

Comment:
I would write it like this:

```python
def pytest_unconfigure(config):
    try:
        xprocess = config._xprocess
    except AttributeError:
        # xprocess fixture was not used   
        pass 
    else:
        xprocess._clean_up_resources()
```

As it is written, you might get an `AttributeError` from code _inside_ `_clean_up_resources()`, which would hide the error.

Reply:
Oops, thanks for the tip :bowtie: